<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>244</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OSBM Updater</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/OSBM_Logo.png</normaloff>:/images/OSBM_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="buttonExport">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Export toolkit to RST</string>
      </property>
      <property name="text">
       <string>&amp;Export</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>://images/export.png</normaloff>://images/export.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="buttonCommand">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Maintain toolkit commands</string>
      </property>
      <property name="text">
       <string>&amp;Commands</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/desktop-32.png</normaloff>:/images/desktop-32.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonToolkit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Maintain toolkits</string>
      </property>
      <property name="text">
       <string>&amp;Toolkits</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/preferences-system.png</normaloff>:/images/preferences-system.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="buttonDatabase">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Open database</string>
      </property>
      <property name="text">
       <string>&amp;Database</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>://images/open-database.png</normaloff>://images/open-database.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>244</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="icon">
    <iconset>
     <normaloff>://images/application-exit.png</normaloff>://images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
